Doc/tools/extensions/c_annotations.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, refcount_filename, stable_abi_file):
         self.stable_abi_data = {}
         with open(stable_abi_file, 'r') as fp:
             for record in csv.DictReader(fp):
-                role = record['role']
+                #role = record['role']
```

Comment:
What is the purpose of leaving `role` uninitialized? I know nothing about the documentation generator so clarification would be great.

Reply:
The role variable was initialized but was never used .Probably at one time it was being used but not anymore.

Reply:
Thank you, I see now. However, small edits not fixing bugs just clutter the changelog so they are not worth bothering.